e2etest/scripts/ci.sh

```diff
@@ -107,44 +92,27 @@ if [ "$PROVIDER_NAME" = "tpm" ] || [ "$PROVIDER_NAME" = "all" ]; then
 fi
 
 if [ "$PROVIDER_NAME" = "pkcs11" ] || [ "$PROVIDER_NAME" = "all" ]; then
-    pushd ${TESTDIR}
+    pushd "${TESTDIR}"
     # This command suppose that the slot created by the container will be the first one that appears
     # when printing all the available slots.
-    SLOT_NUMBER=`softhsm2-util --show-slots | head -n2 | tail -n1 | cut -d " " -f 2`
+    SLOT_NUMBER=$(softhsm2-util --show-slots | head -n2 | tail -n1 | cut -d " " -f 2)
```

Comment:
Just FYI, I expect that with https://github.com/parallaxsecond/parsec/issues/375, finding the slot number won't be needed anymore :)

Reply:
understood, will remove